docs/CONFIGURATION.md

```diff
@@ -15,6 +15,14 @@ Auto MCP accepts configuration via **CLI flags**, **environment variables** (pre
 | Log level                             | `AUTO_MCP_LOGGING_LEVEL`              | `debug`                          |
 | Path to swagger file                  | `AUTO_MCP_SWAGGER_FILE`               | `/server/swagger.json`           |
 | Path to adjustment file (mcp-builder) | `AUTO_MCP_ADJUSTMENTS_FILE`           | `/server/swagger.json`           |
+| Enable OAuth                          | `AUTO_MCP_OAUTH_ENABLED`              | `true`                           |
+| OAuth provider                        | `AUTO_MCP_OAUTH_PROVIDER`             | `github`                         |
```

Comment:
Please add an explicit list of what are the supported providers.
It should be either in CONFIGURATION.md or OAUTH.md.